app/core/worker.py

```
@@ -0,0 +1,30 @@
+async def process_job(job_id: int):
+    await update_job_status(job_id, "processing")
+    await notify_clients(job_id, "processing")
+    try:
+        job = await get_job_by_id(job_id)
+        logging.info(f" Processing job id: {job_id}")
+        await simulate_twilio_call(job["phone_number"], job["message"])
+        logging.info(f" Process Completed for job id: {job_id}")
+        await update_job_status(job_id, "completed")
+        await notify_clients(job_id, "completed")
+    except Exception:
+        logging.error(f"Error processing job {job_id}: {str(e)}")
```

Ruff (0.11.9)

Undefined name `e`

(F821)